integration-cli/docker_cli_help_test.go

```diff
@@ -103,6 +103,9 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 			// Grab just the first word of each line
 			cmd = strings.Split(strings.TrimSpace(cmd), " ")[0]
 			cmds = append(cmds, cmd)
+			if cmd == "daemon" {
```

Comment:
I don't think this should be here - it should be added to `noShortUsage` instead.